core/common/src/main/java/alluxio/cli/CommandUtils.java

```diff
@@ -47,8 +48,10 @@ private CommandUtils() {} // prevent instantiation
       if (cls.getPackage().getName().equals(pkgName + ".command")
           && !Modifier.isAbstract(cls.getModifiers())) {
         // Only instantiate a concrete class
-        Command cmd = CommonUtils.createNewClassInstance(cls, classArgs, objectArgs);
-        commandsMap.put(cmd.getCommandName(), cmd);
+        @Initialized Command cmd = CommonUtils.createNewClassInstance(cls, classArgs, objectArgs);
+        if (cmd != null) {
```

Comment:
At the moment the Checker Framework it's marking this object as ```@UnknownInitialization```, I suspect it is because the method ```createNewClassInstance``` can throw exceptions. From the Checker Framework perspective, It's not clear whether the variable ```cmd``` is "initialized". Below the error message:

```
[ERROR] found   : @UnknownInitialization @NonNull Command
[ERROR] required: @Initialized @NonNull Command
```
Even after marking the object as ```@Initialized``` the Checker Framework still complains that the object may be ```null```. This is why I added the null check. An alternative may be doing the following:

```
@Initialized Command cmd = CommonUtils.createNewClassInstance(cls, classArgs, objectArgs);
assert cmd != null : "@AssumeAssertion(nullness)";
commandsMap.put(cmd.getCommandName(), cmd);
```

What are your thoughts @aaudiber ?